evil-collection.el

```diff
@@ -522,7 +560,9 @@ instead of the modes in `evil-collection-mode-list'."
               reqs (cdr mode)))
       (dolist (req reqs)
         (with-eval-after-load req
-          (require (intern (concat "evil-collection-" (symbol-name m))))
+          (load (expand-file-name (format "modes/%s/evil-collection-%s" m m)
```

Comment:
@justbur , I think something similar to 
```
(load (expand-file-name (format "modes/%s/evil-collection-%s" m m)
                                  evil-collection-base-dir)
                nil t)
```

will have to be done for LOC 575 where is says "(require 'evil-collection-unimpaired)"

Reply:
Never mind, disregard this comment, was just reading and jumping through the multiple branches and I saw that you merged in my git commit that addressed this issue.